src/shared/lib/OncoKbUtils.ts

```diff
@@ -116,11 +116,11 @@ export function generateQueryVariant(hugoSymbol:string,
                                      alterationType?:string): Query
 {
     return {
-        id: generateQueryVariantId(hugoSymbol, tumorType, alteration, mutationType),
-        hugoSymbol,
+        id: generateQueryVariantId(entrezGeneId, tumorType, alteration, mutationType),
+        hugoSymbol: '',
```

Comment:
In this particular case, we are sending request body with POST method which is using `Query` model. In typescript, we need to define a value for each property within that model(this is what I understand. Please correct me if I'm wrong @onursumer ). Similar to what we have currently that we always set entrezGeneId to 0.